CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.13.0-beta-1
+- Unlimited assets regenerated code. (#302)
```

Comment:
Added support for specific account/asset/application queries.

Reply:
Sorry Shant, since we already made the release. Generally we use the PR titles for the release notes. Hopefully we remember to change it to `
Added support for specific account/asset/application queries.` for the stable java sdk release, but that's something to keep in mind when creating the PRs.